gensrc/thrift/DataSinks.thrift

```diff
@@ -299,10 +300,25 @@ struct THiveBucket {
 }
 
 enum THiveCompressionType {
-    SNAPPY = 3,
-    LZ4 = 4,
-    ZLIB = 6,
-    ZSTD = 7,
+    NO_COMPRESSION = 0,
```

Comment:
Reuse the existing definition: TFileCompressType

Reply:
fix in https://github.com/apache/doris/pull/32274